### DISCUSSION TITLE
Rem unit in spacing section

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#348] [Ideas]
Hi,
I usually use rem unit for spacing margin and padding, but it's not available in Astroid for spacing section, do you thing that will be possible to add this unit in a future release?

Thanks,
Lorenzo

![astroid-rem-unit](https://github.com/templaza/astroid-framework/assets/566221/3cfe9303-4a2a-4670-8278-f61079a96d1d)

**Comment by sonvnn** [comment]
Thanks for suggestion! I am working on Astroid 3 with React JS. I love rem unit, too. I will add this to Astroid 3.

Best Regards,
Sonny

**Reply by iorbita** [reply]
wow Astroid 3 🎉

**Reply by Chacapamac** [reply]
Astroid 3, React JS, Super Cool!
I also like Rem...
